OpenRA.Game/Server/Server.cs

```diff
@@ -198,6 +242,15 @@ public Server(List<IPEndPoint> endpoints, ServerSettings settings, ModData modDa
 				}
 			};
 
+			if (Settings.RecordGameReplays)
```

Comment:
This should probably also include `Type == ServerType.Dedicated` - it doesn't make sense to record a second replay for skirmish or campaign games.

Reply:
Added.